setup.py

```diff
@@ -16,35 +16,39 @@
 readme = open('README.rst').read()
 history = open('CHANGES.rst').read()
 
-sphinx_require = 'Sphinx>=1.5.1'
+sphinx_require = 'Sphinx>=3'
 
 tests_require = [
     'pytest-mock>=1.6.0',
-    'pytest-invenio>=1.3.3,<2.0.0',
-    'invenio-accounts>=1.3.0,<2.0.0',
-    'invenio-app>=1.3.0,<2.0.0',
+    'pytest-invenio>=1.4.1',
+    'invenio-accounts>=1.4.3',
+    'invenio-app>=1.3.0',
+    'SQLAlchemy-Continuum>=1.3.6',
+    'psycopg2-binary>=2.8.6',
```

Comment:
moderate: Why are these too needed? Can they just come from `invenio-db`? continuum would be installed if added in the DB extract with `versioning` (see [here](https://github.com/inveniosoftware/invenio-db/blob/master/setup.py#L34)), psycopg should be getting installed already since the `postgresql` extra is already present (due to [here](https://github.com/inveniosoftware/invenio-db/blob/master/setup.py#L31) should be getting in)